cmd/proto-schema-registry/settings.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type settings struct {
+	ZookeeperHost string
```

Comment:
What's ZookeeperHost used for?

Reply:
nothing yet. But it will be used for the kafka stuff coming up